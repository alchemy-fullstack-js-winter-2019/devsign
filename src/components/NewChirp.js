import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.section`
  display: flex;
  flex-direction: column;
  align-items: center;
  img {
    margin-right: 10px;
  }
  .body {
    display: flex;
  }
  button {
    border: 1px solid darkgray;
    border-radius: 5px;
    width: 50%;
    margin-top: 10px;
  }
  textarea {
    border: 1px solid black;
  }
`;

export default function NewChirp() {
  return (
    <Wrapper>
      <div className="body">
        <img src='../assets/prof-pic.png' height="40px" alt="prof pic"/>
        <form >
          <textarea name="chirp"></textarea>
        </form>
      </div>
      <button type="submit">Chirp!</button>
    </Wrapper>
  );
}
