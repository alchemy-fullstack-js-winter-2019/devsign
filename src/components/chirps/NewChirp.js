import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.section`
  display: flex;
  flex-direction: column;
  align-items: center;
  img {
    margin-right: 10px;
  }
  .body {
    display: flex;
  }
  button {
    background: #00bfff;
    color: white;
    padding: 8px;
    border-radius: 7px;
    border: 1 px solid darkgray;
    width: 50%;
    margin-top: 10px;
  }
  textarea {
    border: 1px solid black;
  }
  label {
    color: white;
  }
  span {
    font-size: 1px;
  }
`;

export default function NewChirp() {
  return (
    <Wrapper>
      <div className="body">
        <img src='../assets/prof-pic.png' height="40px" alt="prof pic"/>
        <form>
          <label><span>Post</span>
            <textarea name="chirp"></textarea>
          </label>
        </form>
      </div>
      <button type="submit">Chirp!</button>
    </Wrapper>
  );
}
