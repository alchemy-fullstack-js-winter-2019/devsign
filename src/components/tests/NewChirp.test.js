import React from 'react';
import renderer from 'react-test-renderer';
import NewChirp from '../chirps/NewChirp';

describe('NewChirp', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <NewChirp />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
