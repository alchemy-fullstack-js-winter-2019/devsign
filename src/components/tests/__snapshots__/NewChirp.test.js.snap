// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewChirp matches a snapshot 1`] = `
<section
  className="sc-bdVaJa bUPIVd"
>
  <div
    className="body"
  >
    <img
      alt="prof pic"
      height="40px"
      src="../assets/prof-pic.png"
    />
    <form>
      <label>
        <span>
          Post
        </span>
        <textarea
          name="chirp"
        />
      </label>
    </form>
  </div>
  <button
    type="submit"
  >
    Chirp!
  </button>
</section>
`;
